docs/styleguides/python.md

```diff
@@ -197,11 +197,11 @@ from test_util.helpers import (CI_CREDENTIALS,
 
 * TODO Comment Format
     * Follow the rules on [when to write TODO comments](https://google.github.io/styleguide/pyguide.html?showone=TODO_Comments#TODO_Comments)
-    * Format required will be `TODO (active-developer) [JIRA] - Brief one line description`
+    * Format required will be `#TODO (active-developer) JIRA_ID: Brief one line description`
 
     ```
-    # TODO: (skumaran) DCOS_OSS-952 - Create a flake8 plugin to enforce TODO format.
-    ````
+    #TODO (skumaran) DCOS_OSS-952: Create a flake8 plugin to enforce TODO format.
```

Comment:
Sorry, I don't understand the comment properly.

_And what we use currently_ - is mis-leading, because we don't use any JIRA reference or use a TODO format consistently.

Here is the screenshot base guide style reference which suggest two formats. 


![screenshot 2017-04-17 15 38 19](https://cloud.githubusercontent.com/assets/332330/25107398/fefea8ba-2383-11e7-87e2-30372f48ff82.png)

My proposal and suggestion is that we want to use the JIRA ticket with the active developer name.

Can you please share your desirable format? Please make sure to include the elements
1) TODO term 2) developer name 3) JIRA ID 4) Short Description.